src/mapeo-project.js

```diff
@@ -1572,3 +1610,41 @@ export function baseUrlToWS(baseUrl, projectPublicId) {
 
   return wsUrl.href
 }
+
+/**
+ * @param {import('drizzle-orm/better-sqlite3').BetterSQLite3Database} db
+ * @param {import('./datatype/index.js').MapeoDocTables} table
+ * @returns {Stats}
+ */
+function countWeeks(db, table) {
+  //const timezone = getCurrentTimezoneOffset()
+  /** @type {Array<[string, number]>}*/
+  const values = db
+    .select({
+      week: sql`strftime('%Y-%W', date(${table.createdAt}, 'localtime'))`.as(
+        'week'
+      ),
+      count: count().as('count'),
+    })
+    .from(table)
+    .where(
+      sql`date(${table.createdAt}, 'localtime') >= date('now', '-6 months')`
+    )
+    .groupBy(sql`week`)
+    .orderBy(sql`week`)
+    .all()
+    .map(({ week, count }) => [week, count])
+  const columns = ['week', 'count']
+
+  return { columns, values }
+}
+
+function getCurrentTimezoneOffset() {
```

Comment:
We could make things easier for ourselves and just use the value returned by `new Date().getTimezoneOffset()`? Then in sql `date('now', '${timeoutOffsetInMinutes} minutes')